src/widget/wlibrarysidebar.cpp

```diff
@@ -232,6 +237,9 @@ void WLibrarySidebar::selectChildIndex(const QModelIndex& index, bool selectItem
         return;
     }
     QModelIndex translated = sidebarModel->translateChildIndex(index);
+    if (!translated.isValid()) {
```

Comment:
Is this expected or shall we use VERIFY_OR_DEBUG_ASSERT here?

Reply:
yes, could happen, for example from ... though I don't know how/when concurrent playlist manipulation would happen
https://github.com/mixxxdj/mixxx/blob/e5a2b129f15e637d53523d865f1f11647847059d/src/library/trackset/baseplaylistfeature.cpp#L419-L421